src/LeftNav/LeftNav.less

```diff
@@ -1,26 +1,32 @@
-@import (reference) "../less/index.less";
+@import (reference) "../less/utilities";
 
 .LeftNav {
   .flexbox;
   .shade(color, @primary_blue, 2);
   background: @neutral_off_white;
   user-select: none;
-  .text--light;
+}
+
+.LeftNav--topnav,
+.LeftNav--subnav,
+.LeftNav--collapsed {
```

Comment:
`LeftNav--collapsed` might not be needed here since `LeftNav--topnav` is always assigned to the topnav anyway